src/deepl.ts

```diff
@@ -22,16 +22,52 @@ export class DeepLApi {
       url: "/translate",
       data: qs.stringify({
         auth_key: this.authKey,
-        text: text,
-        target_lang: targetLanguage.toUpperCase()
+        text: text.replace(/<(.*?)>/g, function(i: any, match: string) {
+          if(match.match(/^!subteam/)) {
+            return '';
```

Comment:
I think removing the mention of a user group is reasonable (I don't think the code here should resolve the name of the user group), but eliminating the part can cause a different confusion. How about adding a placeholder like `<ignore>@usergroup</ignore>` or something like that?

Reply:
Fixed this by replacing it with `@[subteam mention removed]`. If the message is translated from English to another language, this placeholder will also be translated. If the origin language is other than English, it will remain in English because DeepL can't translate it.